RUN.md

```diff
@@ -0,0 +1,230 @@
+> Guide to setting up your very own ILP Kit.
+> Estimated time: a couple of hours.
+
+### Table of Contents
+- [Ilp Kit installation](#ilp-kit-setup)
+	- [Get a server instance](#get-a-server-instance)
+	- [Server setup](#server-setup)
+	- [Postgresql setup](#postgresql-setup)
+	- [ILP Kit setup](#ilp-kit-setup)
+- [Domain Setup](#domain-setup)
+	- [Subdomain setup](#subdomain-setup)
+	- [Set up SSL](#set-up-ssl)
+	- [Set up proxy for ILP Kit](#set-up-proxy-for-ilp-kit)
+- [Ilp Kit configuration](#ilp-kit-configuration)
+	- [Register account](#register-account)
+	- [Configure environment](#configure-environment)
+	- [Launch ILP Kit](#launch-ilp-kit)
+	- [Issue some money](#issue-some-money)
+- [Enjoy](#enjoy)
+
+# ILP Kit installation
+
+## Get a server instance
+
+### Buy server
+
+- On [DigitalOcean](https://cloud.digitalocean.com/droplets), click `Create Droplet`.
+- Select the option with 2 Gb of RAM (the $20 one).
```

Comment:
Maybe encourage people to use a referral code that earns them $10 credit and you $25 (once they spend $25). Then offer to give that to them as credit on your ledger 😄